etest/docker/image.py

```diff
@@ -4,11 +4,20 @@
 # etest is freely distributable under the terms of an MIT-style license.
 # See COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+from pathlib import Path
+
 import docker
 
 from etest.docker import common
 
 
+def build(path: Path, *args, **kwargs):
+    """Build a docker image."""
+    path_dir = str(path).rsplit("/", 1)[0]
```

Comment:
We should use the Path methods to do this.  path.basename() should do it if I recall correctly.